src/pages/ocpDetails/detailsChart.tsx

```diff
@@ -46,13 +41,13 @@ type DetailsChartProps = DetailsChartOwnProps &
   DetailsChartDispatchProps &
   InjectedTranslateProps;
 
-const randomId = () => Date.now();
+// const randomId = () => Date.now();
 
-const TooltipFunction = value => {
-  return () => {
-    return <Tooltip id={randomId()}>{`${value.title}`}</Tooltip>;
-  };
-};
+// const TooltipFunction = value => {
```

Comment:
remove this function

Reply:
Yes, this old code can be removed. Thank you